scripts/fetch-schema.js

```diff
@@ -10,12 +10,14 @@ const thegraphConfig = require('../graphql.thegraph.config')
 const exec = promisify(child_process.exec)
 
 function fetchSchema(url, outputFile) {
-  exec(`npx get-graphql-schema --h Origin=https://app.uniswap.org ${url}`)
+  exec(`yarn get-graphql-schema --h Origin=https://app.uniswap.org ${url}`)
```

Comment:
This should use `yarn --silent` to avoid yarn logs, otherwise you'll get extra lines at the top _and_ bottom.

```suggestion
  exec(`yarn --silent get-graphql-schema --h Origin=https://app.uniswap.org ${url}`)
```

Reply:
Beautiful, confirmed the schema was unchanged after fetching. Thanks for the solution.